SampleApp/SearchPinsViewController.swift

```diff
@@ -50,7 +57,15 @@ class SearchPinsViewController: SampleMapViewController, UITextFieldDelegate {
     present(alert, animated: true, completion: nil)
   }
 
-  // MARK:- private
-
+  func selected(_ location: PeliasMapkitAnnotation) {
+    print("Selected \(String(describing: location.title))")
+    searchField.text = location.title
+    location.setTarget(target: self, action: #selector(self.annotationClicked(annotation:)))
+    try? self.removeAnnotations()
+    try? self.add([location])
 
+    animate(toZoomLevel: 10, withDuration: 1.0)
```

Comment:
Nit: maybe we should use `max(10, self.zoom)` so that if the user zooms in further than level 10 or clicks the find me button, the map doesn't animate out?